controllers/operands/kubevirt.go

```diff
@@ -47,7 +47,8 @@ const (
 )
 
 const (
-	primaryUDNNetworkBindingName = "passt"
+	primaryUDNNetworkBindingName    = "passt"
+	managedTapUDNNetworkBindingName = "managedTap"
```

Comment:
I was wrong, we should change the `primaryUDNNetworkBindingName` to `managedTap`, if we want to test passt we just use HCO json patching for kubevirt CR and create manually the NAD, at the end is a development workflow.

Also keeping the same name for the binding is important from the beginning so we don't have to modify VM/VMI to upgrade from managedTap to passt

Reply:
We can rename vars here but dont need to drop old passt support atm, 
the CR can hold both and it will allow smoother QE adaption
same for NAD on CNAO and all, cleaning will be on a follow-up according needs

for now we just need to think about the generic name (unless this also can be done in phases)
lets continue please the design that was started offline

Reply:
switched the primary one to be managedTap
renamed it to have generic name `defaultPodNetworkBinding`